packages/dd-trace/src/plugins/util/exec.js

```diff
@@ -1,6 +1,5 @@
-const { execSync } = require('child_process')
-
 const sanitizedExec = (cmd, options = {}) => {
+  const { execSync } = require('child_process')
```

Comment:
sadly I haven't found a way to do this otherwise: I want to check that I'm calling this function with a `sinon.spy` in https://github.com/DataDog/dd-trace-js/pull/1207/commits/932b3cd8917a583d2e4b5c3af0da657a55fc17e1#diff-64c480332d427d8686183001fd8b6bec833b1e05bcbf633b2bb4cc3708111cf5R7, but if I don't do this, mocha loads the test code "too fast" and makes it so that the `sinon.spy` is too late. 
Open to suggestions for this

Reply:
You can use `proxyquire` to control loaded dependencies.

Reply:
thanks! I'll try that 👍

Reply:
worked like a charm ⭐

Reply:
wait, it works locally but I'm getting errors in the pipeline 😆 . Checking

Reply:
ok now everything's fine. Works like a charm indeed!